### PR TITLE
Add -o command line parameter to specify export format

### PR DESCRIPTION
**Pull request by mconlon17**
Jena3tools modified.  Ready for testing.

### CODE REVIEW

File: jena3tools/src/main/java/org/vivoweb/tools/JenaCli.java [hunk 1]
```diff
@@ -1,9 +1,12 @@
 package org.vivoweb.tools;
 
 import org.apache.commons.lang3.StringUtils;
+import org.apache.jena.vocabulary.RDF;
```
Comment (awoods) [inline jena3tools/src/main/java/org/vivoweb/tools/JenaCli.java]: This `import` can be removed... does not appear to be used.

**Comment by awoods** [comment]
This update allows for export of other RDF serializations, but still requires TRIG for import.
Is there value in also updating the various `RDFDataMgr.read(configurationDataset, inputStream, Lang.TRIG);` calls to read in a configurable `inputFormat`?

**Comment by mconlon17** [comment]
Will take a look.  Import requires a quad format and TRiG was the only supported quad format.  Perhaps there are others that can be supported for input.  But for the use case of "upgrade" it defaults properly, just export (to TriG) and import (from TriG).

For the use case "export," multiple formats make good sense, you may not be interested in the graph info, just want to export to another triple store, or provide as a file.

Regardless, I'll take a look at supported quad formats that might serve as an alternative to TriG.

**Comment by mconlon17** [comment]
jena3tools now supports six output formats.  Three are quad formats -- trig, nq, and jsonld.  Three are triples formats -- nt, rdf, and ttl.  Quad formats write each group with graph information and could potentially be used for input.  Triple formats create a union graph of all triples in all graphs and write the triples only, graph information is not included.

Each of the six output formats have been tested by exporting from a VIVO instance.  Content and configuration files had the correct file type and correct file format in each case.

**Comment by mconlon17** [comment]
Jena2tools now supports five output formats.  Two are quad formats (trig and nq).  Three are triples formats (nt, rdf, ttl).  Note: JSONLD is not a supported output format in Jena 2.

For the configuration, a union model as the union of displayMetaData and userAccounts.  For content, a SQL query is used to identify the graphs in the database, and create a union model of all of them.  The name of each model is displayed as the union is created.

All the output formats have been tested using a VIVO 1.9.3 instance.  Content and configuration files had the correct file type and correct file format in each case.

I also tested jena3tools using a Vitro develop instance.  Worked as expected, producing content and configuration in the specified formats.

Ready for others to test.

**Pull request merged**